src/test/functional/features/list-contact-types.feature

```diff
@@ -0,0 +1,45 @@
+Feature: Contact Types List
+
```

Comment:
I am also wondering if it is possible to do some exploratory testing around trying to get to the pages by entering the url in the box, rather than clicking on the links 

For example: 

If we enter localhost:3300/lists/contactType/id , we should expect to still see an unauthorised message even though we are going straight there rather than navigating through all the links.

Reply:
yeah, an exploratory test should cover this scenario as well. Thanks